src/libraries/System.Formats.Asn1/src/System/Formats/Asn1/Asn1Tag.Accelerators.cs

```diff
@@ -43,7 +43,7 @@ public partial struct Asn1Tag
             new Asn1Tag(ConstructedMask, (int)UniversalTagNumber.OctetString);
 
         /// <summary>
-        ///   Represents the universal class tag for a Null value.
+        ///   Represents the universal class tag for a <see langword="null" /> value.
```

Comment:
I'm not sure this is a correct change.  It's referencing the ASN-NULL, not the .NET-null

Reply:
```suggestion
        ///   Represents the universal class tag for a <c>Null</c> value.
```